af/analysis/analysis/analysis.py

```diff
@@ -942,11 +945,140 @@ def generate_slow_query_summary(conf):
         fi = conf.output_directory / f"db_slow_queries_{role}.html"
         log.info("Writing %s", fi)
         fi.write_text(html)
+        conn.close()
 
     log.info("Writing metrics to node exporter")
     prom.write_to_textfile(node_exporter_path, prom_reg)
 
 
+@metrics.timer("monitor_measurement_creation")
+def monitor_measurement_creation(conf):
+    """Monitor measurements created by fastpath and traditional pipeline
+    to detect and alert on inconsistencies
+
+    Runs in a dedicated thread and writes in its own .prom file
+
+    This is the most important function, therefore it pings the SystemD watchdog
+    """
+    log.info("Started monitor_measurement_creation thread")
+    # TODO: switch to OOID
+
+    INTERVAL = 60 * 5  # half of the watchdog interval in the analysis.service file
+    nodeexp_path = "/run/nodeexp/db_metrics.prom"
```

Comment:
Do we want to have this hardcoded in here?

Reply:
Yes, it's close to where it's being used. monitor_measurement_creation() is meant to be self-contained